### DISCUSSION TITLE
macOS 14 Sonoma

### DISCUSSION THREAD

**Discussion by SkyrilHD** [SkyrilHD/Dell-Exx50-Hackintosh#67] [Show and tell]
Using the Ventura EFI, I was able to Sonoma without any issues. Since there is no GPU acceleration, it is quite unusable. Now, we need to wait until the dortania team adds support for Sonoma :)

![Sonoma on E7250](https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/assets/28839925/d8374bdd-626b-4bbc-b1cf-34f70b83fe8d)

**Comment by SkyrilHD** [comment]
macOS Sonoma Release Candidate is working! However, I experience issues when updating with root patches. Clean install is needed. I will update the EFI shortly, but release will be on the same day as Sonoma gets released

![Bildschirmfoto 2023-09-13 um 16 08 52](https://github.com/SkyrilHD/Dell-E7250-Hackintosh/assets/28839925/46671ceb-d306-4f1e-a499-98b371116b73)

**Reply by SkyrilHD** [reply]
The new EFI is now public on the [master branch](https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/tree/master) and can be tested!

**Reply by mallipudipraveenkumar15-droid** [reply]
can u share that efi for mac os sonoma version

**Reply by mallipudipraveenkumar15-droid** [reply]
newer is. not working perfect

**Reply by mallipudipraveenkumar15-droid** [reply]
getting gpu acceleration no smoothness is there for sonoma

**Comment by samlothphearak** [comment]
![243487261-d8374bdd-626b-4bbc-b1cf-34f70b83fe8d](https://github.com/user-attachments/assets/6c483adc-5ec5-45d9-8095-0dc7a6d5a3d7)
After installing macOS Sonoma on my Dell Latitude E5550, I encountered a graphics issue where macOS only recognizes the GPU as “Display 4 MB” with no hardware acceleration. The user interface is laggy, and there's no smooth animation or performance, which suggests that graphics acceleration is not working.

**Reply by SkyrilHD** [reply]
Follow this guide: https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/#gpu-acceleration-on-ventura-and-newer

The guide is a bit outdated, use OpenCore-Patcher.pkg instead.

**Comment by mallipudipraveenkumar15-droid** [comment]
can u please share updated efi for Dell latitude e5450 for mac os sonoma please share asap emergency

**Reply by SkyrilHD** [reply]
Just use the latest release. Do not forget to install the root patches.
